crates/matrix-sdk/src/config/client.rs

```diff
@@ -127,10 +127,11 @@ impl ClientConfig {
     /// * `path` - The path where the stores should save data in. It is the
     /// callers responsibility to make sure that the path exists.
     ///
-    /// In the default configuration the client will open default
-    /// implementations for the crypto store and the state store. It will use
-    /// the given path to open the stores. If no path is provided no store will
-    /// be opened
+    /// In the default configuration, and if the corresponding features
+    /// (`sled_state_store` and `sled_cryptostore`) are enabled, the client will
+    /// open default implementations for the crypto store and the state store.
+    /// It will use the given path to open the stores. If no path is provided no
```

Comment:
This isn't strictly true, an ephemeral in-memory store will be used as a fallback.